.github/workflows/ci.yaml

```diff
@@ -0,0 +1,40 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+name: ci
+jobs:
+  unit-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [10]
```

Comment:
example of how to run on a matrix of Node.js versions, if there's any wish to do so.